app/javascript/store/index.js

```diff
@@ -95,23 +118,24 @@ const store = new Vuex.Store({
     },
     setLoading: (context, payload) => {
       context.commit('setLoading', payload);
-    }
+    },
   },
   getters: {
     productsAsArray: state => (Object.keys(state.products).map(key => ({ id: key, ...state.products[key] }))),
     totalAmount: (state, getters) => (
       getters.productsAsArray.reduce((acc, product) => acc += product.amount, 0)
     ),
     totalPrice: (state, getters) => (
-      getters.productsAsArray.reduce((acc, product) => acc += (product.price * product.amount), 0)
+      getters.productsAsArray.reduce((acc, product) => acc += product.user_products
```

Comment:
Arrow function should not return assignment  no-return-assign<br>Assignment to function parameter 'acc'       no-param-reassign